src/test/java/uk/gov/hmcts/reform/hmc/client/HearingCodeTest.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.hmc.client;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+class HearingCodeTest {
+
+    @Test
+    void testGetByLabel() {
+        assertEquals(HearingCode.LISTED, HearingCode.getByLabel("LISTED"));
```

Comment:
Could replace string literals with enum string value

Reply:
This test is to check the getByLabel() which takes string as param.